samples/view-store/src/main/java/store/order/view/joined/JoinedCustomerOrdersView.java

```diff
@@ -1,19 +1,19 @@
-package store.view.joined;
+package store.order.view.joined;
```

Comment:
I've decided to keep a separate "view" package, because that's what we want to highlight in this sample.